src/lib.rs

```diff
@@ -109,9 +111,13 @@ where
     /// This starts three background tasks. The swarm, garbage collector and the dht cleanup
     /// tasks run in the background.
     pub async fn new(config: Config) -> Result<Self> {
-        let storage = StorageService::open(config.storage)?;
+        let executor = Executor::new();
+        Self::new0(config, executor).await
```

Comment:
Not a big fan of adding a `new0`. How do you plan on configuring the executor from actyx? I'd suggest adding it to the `Config` instead.

Reply:
Ah I see. You configure it via feature flags, this is only used for the tests.

Reply:
Right, just for tests.